docker/docker-compose.common.yml

```diff
@@ -0,0 +1,45 @@
+version: "3"
+
+services:
+  proxy:
+    image: dockercloud/haproxy
+    environment:
+      - BALANCE=leastconn
+      - EXTRA_GLOBAL_SETTINGS=debug
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    ports:
+      - 80:80
+    networks:
+      - openpension
+
+  database:
+    image: postgres
+    volumes:
+      - ./database:/etc/cron.daily
+      - /var/backups:/var/backups
```

Comment:
Oh ok. So it's a bit confusing because of the folder name (`database`), but the actual data is not in any of those mounts. it's inside the container.
So, i believe what i'm doing right now should work.

Just a question out of curiosity: 
If the db-data folder lives inside the container, how does data persist when you rebuild the containers again? i mean, any change inside the container will vanish when you rebuild, or relaunch the container.